DeploymentCloud/Deployment.DataX/common.parameters.txt

```diff
@@ -32,6 +32,21 @@ databricksClusterNodeType=Standard_DS3_v2
 #Databricks SKU
 databricksSku=premium
 
+# HDInsight cluster version. Please use a whitelisted subscriptionId if you want to deploy version 4.0 (for now).
```

Comment:
looks like this comment is not applicable for the external OSS user?

Reply:
Let's make this comment more generic.

Reply:
removed the inappropriate comment